generated/niscope/niscope/session.py

```diff
@@ -956,7 +956,7 @@ class _SessionBase(object):
     You can specify a subset of repeated capabilities using the Python index notation on an
     niscope.Session repeated capabilities container, and calling set/get value on the result.
     '''
-    _meas_time_histogram_high_time = _attributes.AttributeViReal64(1150028)
+    meas_time_histogram_high_time = _attributes.AttributeViReal64(1150028)
```

Comment:
We should support `timedelta` types for this and for `meas_time_histogram_low_time`; and also for any other properties or method parameters that you're making public. Please check.